.github/workflows/ci.yml

```
@@ -0,0 +1,85 @@
+    steps:
+      # -----------------------------
+      # Checkout
+      # -----------------------------
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      # -----------------------------
+      # Gradle Wrapper Validation
+      # -----------------------------
+      - name: Validate Gradle Wrapper
+        uses: gradle/wrapper-validation-action@v1
```

actionlint (1.7.10)

the runner of "gradle/wrapper-validation-action@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)